extras/profile-and-settings.rst

```diff
@@ -47,6 +47,15 @@ Profile Settings
 
    Consult the `admin documentation <https://admin-docs.zammad.org/en/latest/manage-trigger.html#other-notifications>`_
    for details on how to customize these email notifications.
+   
+   The following *Limit Groups* box allows you to disable notifications for the unchecked 
+   groups. If a group is not checked, a group ticket will not trigger a notification. 
+   You will still get desktop notifications, and email notifications for tickets directly assigned to you.
```

Comment:
@MrGeneration, can you help me verify Zammad's behavior here so that I can improve the clarity of this sentence? I see two possible interpretations:

1. "You wil still get desktop and email notifications for tickets directly assigned to you." (Or, **You will always receive notifications for tickets you own.**)
2. "You will still get all notifications on desktop, and will get email notifications for tickets directly assigned to you." (Or, **Desktop and Email notifications behave differently: Limit Groups applies to the former but not the latter.**)

Is either one of these correct?

Reply:
Heya, thanks for asking back! Number one would be correct here.

You'll always receive notifications based on the notification settings above. You basically only "ignore" updates and creations for tickets that are not yours.

Reply:
Okay, I've applied a commit directly to the PR branch with my desired changes.